src/components/Card/Card.tsx

```diff
@@ -176,11 +176,17 @@ const Card = ({
   const hasBody = Boolean(children);
   const hasFooter = Boolean(footer);
 
+  const handleEventWithAnalytics = useAnalytics();
+  const handleClick = (e: any) =>
+    handleEventWithAnalytics('Card', onClick, 'onClick', e, containerProps || { name: 'Card' });
+  const isDefaultOnClick = onClick === defaultOnClick;
```

Comment:
this is pretty simple logic - could we move it into the StyledContainer styled-component to reduce the API footprint? I know we were doing that before, so I'm not sure why this fixed it as opposed to what was there before